### ISSUE TITLE
Uncaught TYPO3 Exception: An exception occurred while executing 'SELECT * FROM `backend_layout` WHERE (((? = ?) AND (? = ?)) OR...

### ISSUE THREAD

**Issue by starger20** [Gernott/mask#285]
Hello guys,
after a fresh installation of typo3 10.4.2 (composer) I get the following error when I try to open the backend module of mask 5.0.0:

Thu, 14 May 2020 19:24:57 +0200 [CRITICAL] request="997fa74479bf4" component="TYPO3.CMS.Core.Error.ProductionExceptionHandler": Core: Exception handler (WEB): Uncaught TYPO3 Exception: An exception occurred while executing 'SELECT * FROM `backend_layout` WHERE (((? = ?) AND (? = ?)) OR ((`backend_layout`.`pid` = ?) OR (`backend_layout`.`pid` = ?)) OR ((? = ?) AND (`backend_layout`.`pid` = ?))) AND ((`backend_layout`.`deleted` = 0) AND (`backend_layout`.`hidden` = 0) AND ((`backend_layout`.`t3ver_wsid` = 0) AND (`backend_layout`.`t3ver_oid` = 0))) ORDER BY `sorting` ASC' with params [null, 0, 0, 0, null, 0, null, 0, null]:

MySQL server has gone away | Doctrine\DBAL\Exception\DriverException thrown in file /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php in line 106. Requested URL: http://localhost/typo3/index.php?route=%2Fmodule%2Ftools%2FMaskMask&token=--AnonymizedToken-- - {"TYPO3_MODE":"BE","exception":"Doctrine\\DBAL\\Driver\\Mysqli\\MysqliException: MySQL server has gone away in /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php:151\nStack trace:\n#0 /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(906): Doctrine\\DBAL\\Driver\\Mysqli\\MysqliStatement->execute()\n#1 /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/Query/QueryBuilder.php(206): Doctrine\\DBAL\\Connection->executeQuery('SELECT * FROM `...', Array, Array)\n#2 /srv/www/typo3/public/typo3/sysext/core/Classes/Database/Query/QueryBuilder.php(204): Doctrine\\DBAL\\Query\\QueryBuilder->execute()\n#3 /srv/www/typo3/public/typo3/sysext/backend/Classes/View/BackendLayout/DefaultDataProvider.php(200): TYPO3\\CMS\\Core\\Database\\Query\\QueryBuilder->execute()\n#4 /srv/www/typo3/public/typo3/sysext/backend/Classes/View/BackendLayout/DefaultDataProvider.php(54): TYPO3\\CMS\\Backend\\View\\BackendLayout\\DefaultDataProvider->getLayoutData(NULL, Array, NULL)\n#5 /srv/www/typo3/public/typo3/sysext/backend/Classes/View/BackendLayout/DataProviderCollection.php(85): TYPO3\\CMS\\Backend\\View\\BackendLayout\\DefaultDataProvider->addBackendLayouts(Object(TYPO3\\CMS\\Backend\\View\\BackendLayout\\DataProviderContext), Object(TYPO3\\CMS\\Backend\\View\\BackendLayout\\BackendLayoutCollection))\n#6 /srv/www/typo3/public/typo3conf/ext/mask/Classes/Domain/Repository/BackendLayoutRepository.php(82): TYPO3\\CMS\\Backend\\View\\BackendLayout\\DataProviderCollection->getBackendLayoutCollections(Object(TYPO3\\CMS\\Backend\\View\\BackendLayout\\DataProviderContext))\n#7 /srv/www/typo3/public/typo3conf/ext/mask/Classes/Controller/WizardController.php(415): MASK\\Mask\\Domain\\Repository\\BackendLayoutRepository->findAll(Array)\n#8 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Mvc/Controller/ActionController.php(370): MASK\\Mask\\Controller\\WizardController->listAction()\n#9 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Mvc/Controller/ActionController.php(213): TYPO3\\CMS\\Extbase\\Mvc\\Controller\\ActionController->callActionMethod()\n#10 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Mvc/Dispatcher.php(89): TYPO3\\CMS\\Extbase\\Mvc\\Controller\\ActionController->processRequest(Object(TYPO3\\CMS\\Extbase\\Mvc\\Web\\Request), Object(TYPO3\\CMS\\Extbase\\Mvc\\Web\\Response))\n#11 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Mvc/Web/BackendRequestHandler.php(36): TYPO3\\CMS\\Extbase\\Mvc\\Dispatcher->dispatch(Object(TYPO3\\CMS\\Extbase\\Mvc\\Web\\Request), Object(TYPO3\\CMS\\Extbase\\Mvc\\Web\\Response))\n#12 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Core/Bootstrap.php(226): TYPO3\\CMS\\Extbase\\Mvc\\Web\\BackendRequestHandler->handleRequest()\n#13 [internal function]: TYPO3\\CMS\\Extbase\\Core\\Bootstrap->handleBackendRequest(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#14 /srv/www/typo3/public/typo3/sysext/backend/Classes/Http/RouteDispatcher.php(66): call_user_func_array(Array, Array)\n#15 /srv/www/typo3/public/typo3/sysext/backend/Classes/Http/RequestHandler.php(94): TYPO3\\CMS\\Backend\\Http\\RouteDispatcher->dispatch(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#16 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Middleware/SignalSlotDeprecator.php(49): TYPO3\\CMS\\Backend\\Http\\RequestHandler->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#17 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Extbase\\Middleware\\SignalSlotDeprecator->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(TYPO3\\CMS\\Backend\\Http\\RequestHandler))\n#18 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/SiteResolver.php(69): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#19 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\SiteResolver->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#20 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/AdditionalResponseHeaders.php(41): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#21 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\AdditionalResponseHeaders->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#22 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/OutputCompression.php(47): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#23 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\OutputCompression->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#24 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/BackendUserAuthenticator.php(77): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#25 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\BackendUserAuthenticator->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#26 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/BackendRouteInitialization.php(58): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#27 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\BackendRouteInitialization->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#28 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/ForcedHttpsBackendRedirector.php(55): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#29 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\ForcedHttpsBackendRedirector->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#30 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/LockedBackendGuard.php(73): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#31 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\LockedBackendGuard->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#32 /srv/www/typo3/public/typo3/sysext/core/Classes/Middleware/NormalizedParamsAttribute.php(45): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#33 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Core\\Middleware\\NormalizedParamsAttribute->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#34 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(78): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#35 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/AbstractApplication.php(85): TYPO3\\CMS\\Core\\Http\\MiddlewareDispatcher->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#36 /srv/www/typo3/public/typo3/sysext/backend/Classes/Http/Application.php(63): TYPO3\\CMS\\Core\\Http\\AbstractApplication->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#37 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/AbstractApplication.php(97): TYPO3\\CMS\\Backend\\Http\\Application->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#38 /srv/www/typo3/public/typo3/index.php(25): TYPO3\\CMS\\Core\\Http\\AbstractApplication->run()\n#39 /srv/www/typo3/public/typo3/index.php(26): {closure}()\n#40 {main}\n\nNext Doctrine\\DBAL\\Exception\\DriverException: An exception occurred while executing 'SELECT * FROM `backend_layout` WHERE (((? = ?) AND (? = ?)) OR ((`backend_layout`.`pid` = ?) OR (`backend_layout`.`pid` = ?)) OR ((? = ?) AND (`backend_layout`.`pid` = ?))) AND ((`backend_layout`.`deleted` = 0) AND (`backend_layout`.`hidden` = 0) AND ((`backend_layout`.`t3ver_wsid` = 0) AND (`backend_layout`.`t3ver_oid` = 0))) ORDER BY `sorting` ASC' with params [null, 0, 0, 0, null, 0, null, 0, null]:\n\nMySQL server has gone away in /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:106\nStack trace:\n#0 /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php(169): Doctrine\\DBAL\\Driver\\AbstractMySQLDriver->convertException('An exception oc...', Object(Doctrine\\DBAL\\Driver\\Mysqli\\MysqliException))\n#1 /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php(149): Doctrine\\DBAL\\DBALException::wrapException(Object(Doctrine\\DBAL\\Driver\\Mysqli\\Driver), Object(Doctrine\\DBAL\\Driver\\Mysqli\\MysqliException), 'An exception oc...')\n#2 /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(914): Doctrine\\DBAL\\DBALException::driverExceptionDuringQuery(Object(Doctrine\\DBAL\\Driver\\Mysqli\\Driver), Object(Doctrine\\DBAL\\Driver\\Mysqli\\MysqliException), 'SELECT * FROM `...', Array)\n#3 /srv/www/typo3/vendor/doctrine/dbal/lib/Doctrine/DBAL/Query/QueryBuilder.php(206): Doctrine\\DBAL\\Connection->executeQuery('SELECT * FROM `...', Array, Array)\n#4 /srv/www/typo3/public/typo3/sysext/core/Classes/Database/Query/QueryBuilder.php(204): Doctrine\\DBAL\\Query\\QueryBuilder->execute()\n#5 /srv/www/typo3/public/typo3/sysext/backend/Classes/View/BackendLayout/DefaultDataProvider.php(200): TYPO3\\CMS\\Core\\Database\\Query\\QueryBuilder->execute()\n#6 /srv/www/typo3/public/typo3/sysext/backend/Classes/View/BackendLayout/DefaultDataProvider.php(54): TYPO3\\CMS\\Backend\\View\\BackendLayout\\DefaultDataProvider->getLayoutData(NULL, Array, NULL)\n#7 /srv/www/typo3/public/typo3/sysext/backend/Classes/View/BackendLayout/DataProviderCollection.php(85): TYPO3\\CMS\\Backend\\View\\BackendLayout\\DefaultDataProvider->addBackendLayouts(Object(TYPO3\\CMS\\Backend\\View\\BackendLayout\\DataProviderContext), Object(TYPO3\\CMS\\Backend\\View\\BackendLayout\\BackendLayoutCollection))\n#8 /srv/www/typo3/public/typo3conf/ext/mask/Classes/Domain/Repository/BackendLayoutRepository.php(82): TYPO3\\CMS\\Backend\\View\\BackendLayout\\DataProviderCollection->getBackendLayoutCollections(Object(TYPO3\\CMS\\Backend\\View\\BackendLayout\\DataProviderContext))\n#9 /srv/www/typo3/public/typo3conf/ext/mask/Classes/Controller/WizardController.php(415): MASK\\Mask\\Domain\\Repository\\BackendLayoutRepository->findAll(Array)\n#10 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Mvc/Controller/ActionController.php(370): MASK\\Mask\\Controller\\WizardController->listAction()\n#11 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Mvc/Controller/ActionController.php(213): TYPO3\\CMS\\Extbase\\Mvc\\Controller\\ActionController->callActionMethod()\n#12 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Mvc/Dispatcher.php(89): TYPO3\\CMS\\Extbase\\Mvc\\Controller\\ActionController->processRequest(Object(TYPO3\\CMS\\Extbase\\Mvc\\Web\\Request), Object(TYPO3\\CMS\\Extbase\\Mvc\\Web\\Response))\n#13 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Mvc/Web/BackendRequestHandler.php(36): TYPO3\\CMS\\Extbase\\Mvc\\Dispatcher->dispatch(Object(TYPO3\\CMS\\Extbase\\Mvc\\Web\\Request), Object(TYPO3\\CMS\\Extbase\\Mvc\\Web\\Response))\n#14 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Core/Bootstrap.php(226): TYPO3\\CMS\\Extbase\\Mvc\\Web\\BackendRequestHandler->handleRequest()\n#15 [internal function]: TYPO3\\CMS\\Extbase\\Core\\Bootstrap->handleBackendRequest(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#16 /srv/www/typo3/public/typo3/sysext/backend/Classes/Http/RouteDispatcher.php(66): call_user_func_array(Array, Array)\n#17 /srv/www/typo3/public/typo3/sysext/backend/Classes/Http/RequestHandler.php(94): TYPO3\\CMS\\Backend\\Http\\RouteDispatcher->dispatch(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#18 /srv/www/typo3/public/typo3/sysext/extbase/Classes/Middleware/SignalSlotDeprecator.php(49): TYPO3\\CMS\\Backend\\Http\\RequestHandler->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#19 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Extbase\\Middleware\\SignalSlotDeprecator->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(TYPO3\\CMS\\Backend\\Http\\RequestHandler))\n#20 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/SiteResolver.php(69): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#21 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\SiteResolver->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#22 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/AdditionalResponseHeaders.php(41): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#23 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\AdditionalResponseHeaders->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#24 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/OutputCompression.php(47): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#25 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\OutputCompression->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#26 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/BackendUserAuthenticator.php(77): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#27 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\BackendUserAuthenticator->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#28 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/BackendRouteInitialization.php(58): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#29 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\BackendRouteInitialization->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#30 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/ForcedHttpsBackendRedirector.php(55): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#31 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\ForcedHttpsBackendRedirector->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#32 /srv/www/typo3/public/typo3/sysext/backend/Classes/Middleware/LockedBackendGuard.php(73): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#33 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Backend\\Middleware\\LockedBackendGuard->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#34 /srv/www/typo3/public/typo3/sysext/core/Classes/Middleware/NormalizedParamsAttribute.php(45): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#35 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(172): TYPO3\\CMS\\Core\\Middleware\\NormalizedParamsAttribute->process(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest), Object(class@anonymous))\n#36 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/MiddlewareDispatcher.php(78): class@anonymous->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#37 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/AbstractApplication.php(85): TYPO3\\CMS\\Core\\Http\\MiddlewareDispatcher->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#38 /srv/www/typo3/public/typo3/sysext/backend/Classes/Http/Application.php(63): TYPO3\\CMS\\Core\\Http\\AbstractApplication->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#39 /srv/www/typo3/public/typo3/sysext/core/Classes/Http/AbstractApplication.php(97): TYPO3\\CMS\\Backend\\Http\\Application->handle(Object(TYPO3\\CMS\\Core\\Http\\ServerRequest))\n#40 /srv/www/typo3/public/typo3/index.php(25): TYPO3\\CMS\\Core\\Http\\AbstractApplication->run()\n#41 /srv/www/typo3/public/typo3/index.php(26): {closure}()\n#42 {main}"}

This occurs with a fresh and also with a migrated (from V9) typo3 installation.
Everything else seems to work.
I have no clue about the meaning of this error message, so I decided to ask here.

My system:
Opensuse Tumbleweed
Apache server 2.4.43-1.1
MariaDB 10.4.12-7.1
Php 7.4.5-1.2 (with php-fpm)

Thanks in advance...

**Comment by starger20** [comment]
Meanwhile I tried a downgrade of Php (7.2.5). That failed.
Switching the database to sqlite worked, but sadly this is not an option for my project.

After that I changed the following file:
.../typo3/public/typo3conf/ext/mask/Classes/Domain/Repository/BackendLayoutRepository.php, line 81:
`
$dataProviderContext = $this->backendLayoutView->createDataProviderContext()->setPageTsConfig($pageTsConfig);
return $backendLayouts;
$backendLayoutCollections = $this->backendLayoutView->getDataProviderCollection()->getBackendLayoutCollections($dataProviderContext);`

So basically cutting the function with a return.
The backend module appears now.
My programming knowledge is not great. So this does not fix anything, but maybe gives a guess about the origin of the error.

**Comment by biesior** [comment]
I can confirm this problem 

MacOS Mojave
PHP 7.4.6 and also 7.2.22

Backend layout created at `PID:1` and working in BE w/out problems, mask ext `general.backendlayout_pids = 1`. When interupt query within `findAll($pageTsPids = [])` method of `BackendLayoutRepository` modul works w/out SQL error but of course it doesn't find any backend layouts so we have _No Backendlayoyt found_ warning

SQL problem rises at line:
`$backendLayoutCollections = $this->backendLayoutView->getDataProviderCollection()->getBackendLayoutCollections($dataProviderContext);` 
in mentioned repo, debugging showed that all fields in $dataProviderContext has null values, unfortunatelly I also cannot figure it out what's wrong.

**Comment by biesior** [comment]
FYI: I installed separate instance of `5.7.29 - MySQL Community Server` and problem disappears, which means it's connected with `10.4.6-MariaDB`. This is partial solution for me, as I would like to stay at MariaDB (lot of other projects and faster). At least we have some clue, let's hope that Masked People can resolve this problem.   ;)

**Comment by martinhummer** [comment]
Stumbled across this issue moments ago, thanks for further investigation.

TYPO3 10.4
PHP 7.2
MariaDB 10.4.10

**Comment by ByteBandito** [comment]
We also have this issue on 9.5 and 10.4. Any solutions yet, but changing to MariaDB?

**Comment by nhovratov** [comment]
This might be fixed with MariaDB 10.4.14. See: https://jira.mariadb.org/browse/MDEV-20261

**Comment by nhovratov** [comment]
Duplicate of #246

**Comment by marcelmarty** [comment]
Can confirm this problem too. MariaDB 10.4.11 on XAMPP

**Comment by ilomedia** [comment]
Same problem here with MariaDb 10.4.13, guess we have to wait for MariaDb bugfix

**Comment by nhovratov** [comment]
It should work on 10.5 stable version already as i have heard.

**Comment by ilomedia** [comment]
10.5 is not available to Homebrew yet, as a temporary workaround i downgraded to 10.3

**Comment by nhovratov** [comment]
Can someone test this with the now released 10.4.14 version?

**Comment by nhovratov** [comment]
Fixed in latest mariadb version.
